src/components/vm/consoles/VncConsole.jsx

```diff
@@ -55,6 +55,7 @@ export const VncConsole = ({
     repeaterID = '',
     vncLogging = 'warn',
     consoleContainerId,
+    onConnected = (element) => {},
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2254-9e3d340a-20250805-063134-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)